test/run-spec-wasm2c.py

```diff
@@ -644,7 +644,10 @@ def main(args):
 
             # Run the resulting binary
             if options.run:
-                utils.Executable(main_exe, forward_stdout=True).RunWithArgs()
+                error = utils.Executable(main_exe, forward_stdout=True).RunWithArgsAndError()
```

Comment:
Given that we're printing `error` out now (which includes the stdout from the process, if captured), can we remove the `forward_stdout=True` from this line?

That will prettify the new test's output a little; instead of `None` (for the stdout that went directly to the terminal), it'll say `0/1 tests passed.` (since stdout is now being captured in the Error).

Reply:
yeah, we don't like how this test runner is set up. unfortunately we do not have the capacity to re-engineer it to do things properly.

because in this case, we were trying to keep the stdout in stdout but also get the underlying status code into stderr. (since this runs in the context of `run-spec-wasm2c.py` we can't match the status code with the usual status code matching directive `ERROR`, but nothing stops us from shoving it in stderr.)

Reply:
Fair enough on the larger point, but if we get rid of the `, forward_stdout=True` from this line, the output from the new test becomes a little prettier, none of the old tests change, and stdout still goes to stdout on successful runs. Everything else lgtm.